@navikt/core/react/src/form/error-summary/ErrorSummary.tsx

```diff
@@ -100,7 +100,10 @@ export const ErrorSummary = forwardRef<HTMLDivElement, ErrorSummaryProps>(
         </Heading>
         <BodyShort as="ul" size={size} className="navds-error-summary__list">
           {React.Children.map(children, (child) => {
-            return <li key={child?.toString()}>{child}</li>;
+            if (!child) {
```

Comment:
https://codesandbox.io/s/sharp-gates-dpsgv8?file=/src/App.js

Kan erstatte `!child` med [isValidElement](https://react.dev/reference/react/isValidElement) for å være enda sikrere på at riktig child blir brukt

Reply:
Done ✅